MPC-ODP/text.tex

```diff
@@ -141,198 +154,325 @@ \subsection{Zvláštní kategorie osobních údajů}
+\item je přítomna nezůčastněná osoba, která kontroluje že nedochází k~porušení zákona,
+\item je vypracován protokol (video, fotografie) který všichni zúčastnění podepisují.
 \end{itemize}
-Aby přistihly zločince se zapnutím PC a přihlášením do vzdálené služby - dá se přizvat znalec pro specifickou
-činnost se specifickým vybavením - musí se dodržovat specifické postupy aby nebyly důkazy znehodnocovány - zapečetění, zabalení do pytle za přítomnosti nezúčastněné osoby.
 
-\subsubsection*{Výdání a odnětí věci}
-Nástroj pro zajištění věci od člověka - každý má Ediční povinnost (na požádání musí vydat drženou věc) - OČTŘ ho vyzvou k předložení věci - pokud odmítně věc vydat může mu být odejmuta (pořádkové opatřené) stačí rozhodnutí státního zástupce - při odejmutí věci by měla být přítomna nezúčastněná osoba - pořizuje se protokol - osobě se dá potvrzení o odejmutí - pokud jsou na zařízení data o kterých je povinná mlčenlivost (utajované informace, advokátní tajemství) je specifický postup - pouze věci né data - ovšem pokud je vydán např telefon tak naněm může být provedena forenzní analýza a tedy je možno se dostat k datům uloženým na zařízení
+Znalec i vyšetřovatel musí dodržet specifické postupy aby nedošlo ke~znehodnocení důkazů (zapečetění, zabelení do~pytle).
 
-\subsubsection*{Osobní prohlídka}
-domnívám (jako OČTŘ) se že daná osoba má u sebe věc důležitou pro TŘ, ale nevím to jistě -> zahájím osobní prohlídku - na základě rozhodnutí soudu nebo státního zástupce - pokud je to neopakovatelný úkon můžu osobní prohlídku provéset i bez příkazu (zatknu osobu co utíká z místa činu a mám podezření že u sebe má zbraň se kterou páchal TČ) musím pak ale souhlas zpětně získat - je to zhojitelná vada neučinného důkazu - úkonu by měl předcházet předchozí výslech (jako u domovní prohlídky) což bych osobu měl požádat zda věč/předmět u sebe má a zda mi ho nevydá.
+\subsubsection{Vydání a odnětí věci}
 
-\subsubsection*{Ohledání věci}
-pozorování a sbírání informací za účelem objasnění věci - protokol co bylo vypozorováno a jak - typicky při domovních a osobních prohlídkách - na místě najdu spuštěný počítač a na místě chci provést jeho ohledání - v takovém případě s kamerou nebo foťákem provádím záznam toho co jsem objevil.
+OČTŘ vyzývají k~předložení věci (při~odmítnutí může být odejmuta po~svolení státního zástupce).
+Při~odejmutí musí být přítomna nezávislá osoba, musí být sepsán protokol a osoba musí dosta potvrzení o~odejmutí.
 
-Nemůžu provést obecné ohledání věci - např. když ohledávám na místě zaplé PC a bude tam probíhající komunikace a já bych ji chtěl sledovat (odposlech má větší zásah do práv), tak to nemůžu udělat jen na základě ohledání věci (generovalo by to neúčinný důkaz) - avšak policie si proto může předem připravit příkazy- například existují příkazy k domovní prohlídce, odposlechu a sledování osob a věcí a tím pádem může získávat všechno na místě.
+% TODO Je to třeba? Nešlo by to jasněji?
```

Comment:
JFYI osobně si myslím že na naučení stačí ta obecná podkapitola. Tady tu otázku jsem dostal od dvou lidí. Pavlex ji měl spracovanou obecně a ne tak do detailu a pak Vikča tam měla tyhle detaily. Nechal jsem tady obě verze, že někdo to může chtít do uplného detailu.